### DISCUSSION TITLE
今天编译AX6选了Turbo ACC插件，编译出来就是没有

### DISCUSSION THREAD

**Discussion by zjwxh** [Boos4721/openwrt#78] [Bugs/问题反馈]
今天晚上编译AX6选了Turbo ACC插件，编译了几次出来就是没有，用大雕的库编译正常，不知道是否是大佬源码问题

**Comment by Boos4721** [comment]
clean toolchain and reselete config

**Reply by zjwxh** [reply]
谢谢了，下班回去试试

**Comment by Deleted-Account** [comment]
实际情况应该是 luci-app-turboacc 有编译进去
但是 LuCI 界面没显示

刚编译的 AX6 固件刷进去后，Software 列表下，看到是有 luci-app-turboacc 的

尝试直接访问 路由IP/cgi-bin/luci/admin/network/sfe
提示
No page is registered at '/admin/network/sfe'.
If this url belongs to an extension, make sure it is properly installed.
If the extension was recently installed, try removing the /tmp/luci-indexcache file.

**Reply by Boos4721** [reply]
https://<IP>/cgi-bin/luci/admin/network/turboacc

**Reply by Deleted-Account** [reply]
> cgi-bin/luci/admin/network/turboacc

http://XXXXX/cgi-bin/luci/admin/network/turboacc

No page is registered at '/admin/network/turboacc'.
If this url belongs to an extension, make sure it is properly installed.
If the extension was recently installed, try removing the /tmp/luci-indexcache file.

---

the luci-app-turboacc is in software list.
(language package luci-i18n-turboacc-zh-cn is in list too)

**Reply by Boos4721** [reply]
> > cgi-bin/luci/admin/network/turboacc
> 
> http://XXXXX/cgi-bin/luci/admin/network/turboacc
> 
> No page is registered at '/admin/network/turboacc'. If this url belongs to an extension, make sure it is properly installed. If the extension was recently installed, try removing the /tmp/luci-indexcache file.
> 
> the luci-app-turboacc is in software list. (language package luci-i18n-turboacc-zh-cn is in list too)

![iShot2022-02-23 11 20 54](https://user-images.githubusercontent.com/43845654/155256624-0b9acd5d-4c36-4a7e-aaf0-181a0bb6a71d.jpg)

**Comment by Deleted-Account** [comment]
T—T
![image](https://user-images.githubusercontent.com/48331538/155257983-08cd1770-1a31-4565-a70a-6f4ef6d84ffc.png)
I've no idea what's going on...

**Comment by Boos4721** [comment]
**FIXED**

**Reply by zjwxh** [reply]
今天晚上我编译确实正常了，谢谢大佬修复。另外那个cpu超频的组件没看到了，但是cpu实际在动态超频中，最高有2.2G
